app/views/unattended/snippets/_pxegrub2_chainload.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: snippet
+name: pxegrub2_chainload
+%>
+menuentry "Chainload into bootloader on first disk" {
+  insmod part_msdos
+  insmod chain
+  chainloader (hd0,0)+1
```

Comment:
This does not work for me with RedHat 7. The disk is called (hd0,gtp1) and the "+1" does not work. Haven't tried it further.